web/app.js

```diff
@@ -41,6 +41,12 @@ import { SecondaryToolbar } from 'pdfjs-web/secondary_toolbar';
 import { Toolbar } from 'pdfjs-web/toolbar';
 import { ViewHistory } from 'pdfjs-web/view_history';
 
+var {
```

Comment:
In contrast to the commit message, this does not actually remove `import * as pdfjsLib from 'pdfjs-web/pdfjs';`. Why not, i.e., why is this particular instance being done this way instead of altering the `*` import like the other occurrences in this diff?